assets/src/components/propertiesPanel/vehiclePropertiesPanel.tsx

```diff
@@ -13,13 +14,12 @@ import {
 } from "../../realtime"
 import { Route, RouteId, Shape } from "../../schedule"
 import Map from "../map"
-import PropertiesList from "../propertiesList"
+import PropertiesList, { vehicleProperties } from "../propertiesList"
 import BlockWaiverList from "./blockWaiverList"
 import CrowdingDiagram from "./crowdingDiagram"
 import Header from "./header"
 import HeadwayDiagram from "./headwayDiagram"
 import TabPanels, { TabMode } from "./tabPanels"
-import { useNearestIntersection } from "../../hooks/useNearestIntersection"
```

Comment:
@skyqrose what's up with this coming out?

Reply:
The imports were out of order. It was inserted above. Not sure why CI didn't catch it before.